Sources/Commandant/Command.swift

```diff
@@ -117,6 +117,9 @@ public final class CommandRegistry<ClientError: Error> {
 }
 
 extension CommandRegistry {
+	public func main(defaultVerb: String, completionHandler:() -> Void, errorHandler: (ClientError) -> ()) -> Never  {
```

Comment:
This addition seems unnecessary to me. It could be added as a parameter to the next initializer instead.